static/js/components/VideoPlayer_test.js

```diff
@@ -303,30 +303,30 @@ describe("VideoPlayer", () => {
       }
     })
   })
-  ;[false, true].forEach(function(useYoutube) {
+  ;[false, true].forEach(function(useYouTube) {
     it(`checkYouTube ${expect(
-      useYoutube
-    )} be called if useYoutube is ${String(useYoutube)}`, async () => {
-      const wrapper = renderPlayer({ useYouTube: useYoutube })
+      useYouTube
+    )} be called if useYoutube is ${String(useYouTube)}`, async () => {
+      video.is_public = useYouTube
+      video.youtube_id = "fakeId"
+      const wrapper = renderPlayer()
       const checkStub = sandbox.stub(wrapper.instance(), "checkYouTube")
       wrapper.instance().componentDidMount()
-      sinon.assert.callCount(checkStub, useYoutube ? 1 : 0)
+      sinon.assert.callCount(checkStub, Number(useYouTube))
     })
   })
```

Comment:
on this test, I might suggest that we make it fully parameterized, to test that things get called as we expect with all possible combinations of `is_public === true` and `video.youtube_id !== null`

Reply:
✅